### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
-bower_components
+bower_components*
```
Comment (stramel) [inline .gitignore]: Need to ignore `bower-*.json` as well here.

File: .travis.yml [hunk 2]
```diff
@@ -1,19 +1,23 @@
 language: node_js
-node_js: '6'
+node_js: stable
 addons:
   firefox: latest
+  sauce_connect: true
```
Comment (stramel) [inline .travis.yml:5]: Is this needed now?

File: bower.json [hunk 3]
```diff
@@ -20,12 +20,25 @@
...
+    "polymer": "Polymer/polymer#^2.0.0-rc.1"
   },
   "devDependencies": {
-    "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
-    "iron-test-helpers": "polymerelements/iron-test-helpers#^1.0.0",
-    "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0"
+    "iron-component-page": "PolymerElements/iron-component-page#2.0-preview",
+    "iron-test-helpers": "PolymerElements/iron-test-helpers#2.0-preview",
+    "web-component-tester": "^6.0.0-prerelease.6",
```
Comment (stramel) [inline bower.json:28]: Can we specify `Polymer/web-component-tester#^6.0.0-prerelease.6`?

File: bower.json [hunk 4]
```diff
@@ -20,12 +20,25 @@
...
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "polymer": "Polymer/polymer#^1.1.0"
+      },
+      "devDependencies": {
+        "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
+        "iron-test-helpers": "PolymerElements/iron-test-helpers#^1.0.0",
+        "web-component-tester": "^4.0.0",
```
Comment (stramel) [inline bower.json:39]: Can we specify `Polymer/web-component-tester#^5.0.0`?

File: test/index.html [hunk 5]
```diff
@@ -22,8 +22,8 @@
   <script>
 
     WCT.loadSuites([
-      'basic.html',
-      'basic.html?dom=shadow'
+      'basic.html?dom=shadow',
```
Comment (stramel) [inline test/index.html:25]: Do we need to explicitly add `?dom=shadow` here?

File: .travis.yml (hunk 2 above)
Comment (keanulee) [reply .travis.yml:5]: travis config files are maintained by @tedium-bot

File: bower.json (hunk 3 above)
Comment (keanulee) [reply bower.json:28]: wct is publicly listed on bower, so specifying the repo is optional

File: test/index.html (hunk 5 above)
Comment (keanulee) [reply test/index.html:25]: `?dom=shadow` tests native SD with Polymer 1, and `wc-shadydom=true&wc-ce=true` tests polyfills with Polymer 2.

File: .travis.yml (hunk 2 above)
Comment (stramel) [reply .travis.yml:5]: Hmm, I haven't seen this in any of the other PolymerElement repos. Just thought I would call it out as an inconsistency.

File: bower.json (hunk 3 above)
Comment (stramel) [reply bower.json:28]: All these are published publicly on bower. Seems like for consistency we would want to list this the same way.

File: test/index.html (hunk 5 above)
Comment (stramel) [reply test/index.html:25]: ah, yeah, you are right. Sorry.

**Pull request merged**